src/astroFactory.js

```diff
@@ -0,0 +1,81 @@
+import { MeshBuilder } from "@babylonjs/core/Meshes/meshBuilder"
+import { Texture } from "@babylonjs/core/Materials/Textures/texture"
+import { StandardMaterial } from "@babylonjs/core/Materials/standardMaterial"
+import { Scalar } from "@babylonjs/core/Maths/math.scalar"
+import { Animation } from "@babylonjs/core/Animations/animation"
+
+import logger from "./logger";
+
+import distortTexture from "../assets/textures/distortion.png";
+import rockTextureN from "../assets/textures/rockn.png";
+import rockTexture from "../assets/textures/rock.png";
+
+class AstroFactory {
+    static createPlanet(opts, scene) {
+        let planet = MeshBuilder.CreateSphere(opts.name, { diameter: 1 }, scene);
+        let mat = new StandardMaterial(planet.name + "-mat", scene);
+        mat.diffuseColor = mat.specularColor = mat.emissiveColor = opts.color;
```

Comment:
Did you really mean to set all of those colors to be the same?

Reply:
Probably not, no, but I think I was probably too lazy to add the other colors to the options object at the time :D

I guess if I leave off the emissive color then I could also remove the logic in startScene.js that adds the planets to the excludedMesh list of the GlowLayer too then, right?